plugin/helpers.go

```
@@ -42,3 +47,13 @@ type Helpers interface {
 type HelpersImpl struct {
 	API API
 }
+
+func ensureServerVersion(required string, actual string) error {
+	requiredVersion, _ := semver.Make(required)
```

Please use `MustParse` here, so we don't ignore the error.